test/src/tests/runner.go

```diff
@@ -10,37 +10,44 @@ import (
 	"tests/generated/report"
 )
 
+const testServerPath = "../../../node_modules/@microsoft.azure/autorest.testserver"
+
 func main() {
-	server, err := startServer()
+	err := startServer()
 	if err != nil {
-		fmt.Printf("Error! %v\n", err)
+		fmt.Printf("Error starting server: %v\n", err)
```

Comment:
If this fails, can tests run? If not, we should terminate the program here.

Reply:
I was thinking about this too, I will change this to panic.